src/backend/distributed/citus--9.4-1--9.5-1.sql

```diff
@@ -0,0 +1,4 @@
+-- citus--9.4-1--9.5-1
```

Comment:
This file should be placed in the sql directory.
src/backend/distributed/ **sql/** citus--9.4-1--9.5-1.sql

Also we should introduce a similarly empty downgrade file at
src/backend/distributed/sql/citus--9.5-1--9.4-1.sql